scripts/test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Do we really need this script? Isn't it enough to start the container and then run the test manually?

Reply:
It's not required, but makes things a lot easier

Reply:
You can of course test without it, but I'd prefer using it over starting everything manually

Reply:
Okay, in this case I suggest adding a `sudo` in front of every `docker` command, because sudoing the dart command can have side effects.

Reply:
Normally you should be able to run any docker command without `sudo`, because you should be in the `docker` group (https://docs.docker.com/engine/install/linux-postinstall/). Also `sudo` may not be install on every machine (but almost any machine has it).

Reply:
Ok seems it's not the default. Not sure what to do now

Reply:
Maybe the script should check if the user running the script is not the root user and then use `sudo` on the docker commands so we make sure that the tests are not running using root.

Reply:
See warning in [install-section](https://wiki.archlinux.org/index.php/Docker#Installation). Being in the docker group may be a security risk thus not every system will be setup this way. I see two options:
- make it as you want and everyone who has another setup will either customize the script locally or not use it
- or, setup for the most common case.. I guess `sudo docker ...` should work on most of the systems

Reply:
I'll go for the second option